src/views/notifications/OrderStatusNotification.tsx

```diff
@@ -62,10 +62,11 @@ export const OrderStatusNotification = ({
   const { assetId } = orEmptyObj(marketData);
   const logoUrl = useAppSelector((s) => getAssetImageUrl(s, assetId));
   const { equityTiersLearnMore } = useURLConfigs();
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
```

Comment:
can we add a comment as to why it's disabled here

Reply:
tbh it's mostly just me being super extra careful to not cause errors.